lib/protobuf/nats/server.rb

```diff
@@ -55,7 +55,12 @@ def enqueue_request(request_data, reply_id)
         end
 
         # Publish an ACK to signal the server has picked up the work.
-        nats.publish(reply_id, ::Protobuf::Nats::Messages::ACK) if was_enqueued
+        if was_enqueued
+          nats.publish(reply_id, ::Protobuf::Nats::Messages::ACK)
+        else
+          # pending client support being rolled out
+          # nats.publish(reply_id, ::Protobuf::Nats::Messages::NACK)
```

Comment:
Existing clients should drop a NACK fail to continue even if a NACK is given since no ACK is received. You should be able to roll this out as a non-breaking change.